app/players/elves/elf.py

```diff
@@ -0,0 +1,13 @@
+from app.players.player import Player
+from abc import ABC
```

Comment:
it is not necessary

Reply:
from abc import ABC is necessary!!! Code will be work without that. But today with Danya we discussed and he said that it is a good practice to write because Elf is abstract class and you can not remember it in big project. 
But when you see ABC,  you know that it is abstract. I think it is right.